### ISSUE TITLE
Design Refresh

### ISSUE THREAD

**Issue by Dignifiedquire** [elixir-lang/ex_doc#150]
I really enjoy the integration of elixir, with the automated documentation generator but felt myself wishing for a more up to date design of this documentation. So I opened up Photoshop and started designing. This is what I've come up with so far, I would love some feedback on this before moving onto the details and implementation.

A few things should be noted, this design expects the removal of iframes. I think that would be good improvement, but if there are strong reasons for keeping the iframe layout I could work around that.

![redesign01](https://cloud.githubusercontent.com/assets/790842/4030443/076a6136-2c5d-11e4-9114-f77d560844fc.png)

**Comment by milmazz** [comment]
@Dignifiedquire Your proposal looks really nice! Any recent progress on this issue? I can help you if you want, just let me know.

**Comment by Dignifiedquire** [comment]
@milmazz yes please, I haven't had any time to work in this recently, the most pressing task on the code is to incorporate all the changes that have been made since I started work on this. Feel free to ping me if you have any questions (warning the code is in a bit of a mess :( )

**Comment by milmazz** [comment]
@Dignifiedquire Great, I've reviewed  your `design` branch and the amount of changes is huge! So, do you mind if I try to split your proposal in a series of PR? I ask because there are some changes that are not related with the design (e.g. `:fenced_code` options for `Hoedown`, among others).

**Comment by Dignifiedquire** [comment]
sure go ahead..there was one thing leading to the other when trying to implement it and so it grew and grew...

**Comment by Dignifiedquire** [comment]
@josevalim after all the great work from @milmazz I want to pick this up, just wanted some feedback from you if you like the direction of the design.

**Comment by josevalim** [comment]
That's awesome, @Dignifiedquire! The original design looks excellent. At first look I wouldn't add nor remove anything from it. :) Do you want me to check something in particular?

**Comment by Dignifiedquire** [comment]
No, just wanted to be sure not to go in the wrong direction :)

**Comment by josevalim** [comment]
It is readable, it is clean and elegant. Go for it!

**Comment by Dignifiedquire** [comment]
New branch https://github.com/Dignifiedquire/ex_doc/tree/new-design new luck

Current state: 
![screen shot 2015-06-11 at 20 57 31](https://cloud.githubusercontent.com/assets/790842/8115559/8472c7a0-107c-11e5-81ab-1a5fa47d38e4.png)

**Comment by josevalim** [comment]
Looking great! :D

**Comment by josevalim** [comment]
Hey @Dignifiedquire, how is this working? Is there anything we can do to help? Once the design is ready, we can probably help with integrating it into exdoc.

**Comment by Dignifiedquire** [comment]
Hey @josevalim sorry, the time I was hoping to have didn't turn out to be as much as I hoped, I'll try to get to it as soon as I can

**Comment by josevalim** [comment]
Don't be sorry! Just let us know if we can help with anything!

**Comment by Dignifiedquire** [comment]
Closing in favor of #275
